### ISSUE TITLE
User Login Page

### ISSUE THREAD

**Issue by BorderITS** [borderitsolutions/amadaa#3]

**Comment by lcabrini** [comment]
As soon as I get login working I'll create an empty placeholder dashboard.

**Comment by BorderITS** [comment]
Ok so that the UI designers can work on

**Comment by BorderITS** [comment]
error code 


Page not found (404)
Request Method: 	GET
Request URL: 	http://localhost:8000/accounts/profile/

Using the URLconf defined in amadaa.urls, Django tried these URL patterns, in this order:

    ^$ [name='dashboard']
    ^admin/
    ^ ^login/$ [name='login']
    ^ ^logout/$ [name='logout']
    ^ ^password_change/$ [name='password_change']
    ^ ^password_change/done/$ [name='password_change_done']
    ^ ^password_reset/$ [name='password_reset']
    ^ ^password_reset/done/$ [name='password_reset_done']
    ^ ^reset/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/$ [name='password_reset_confirm']
    ^ ^reset/done/$ [name='password_reset_complete']

The current URL, accounts/profile/, didn't match any of these.

You're seeing this error because you have DEBUG = True in your Django settings file. Change that to False, and Django will display a standard 404 page.

**Comment by lcabrini** [comment]
It looks like you don't have the current amadaa. Could you please git pull and try again?

**Comment by BorderITS** [comment]
ok

**Comment by lcabrini** [comment]
No, I know what the problem is. But we are hijacking a different issue, so I'm doing the right thing and opening a new issue here: #117

**Comment by lexenington** [comment]
For some reason my JavaScript files working with the dashboard page does not load. Not even jquery. This has kept me delayed

**Comment by lcabrini** [comment]
Jquery and bootstrap should already be loaded. Please check the page source.

**Comment by lexenington** [comment]
Java Script filesFile have been loaded but it give 404 error saying they could not load
